### PR TITLE
Add TableCaption to styles.xml

### PR DESCRIPTION
**Pull request by iandol**
See #10053 for details.

### CODE REVIEW

File: data/odt/styles.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@ xmlns:of="urn:oasis:names:tc:opendocument:xmlns:of:1.2"
 xmlns:xhtml="http://www.w3.org/1999/xhtml"
 xmlns:grddl="http://www.w3.org/2003/g/data-view#"
 xmlns:officeooo="http://openoffice.org/2009/office"
-xmlns:tableooo="http://openoffice.org/2009/table"
+xmlns:tableooo="http://openoffice.org/2009/
```
Comment (jgm) [inline data/odt/styles.xml:27]: Why tihs change? It seems unrelated.

Comment (iandol) [reply data/odt/styles.xml]: That was a mistake, I didn't intend that edit... Reverted
